pkg/cmd/root/root_test.go

```diff
@@ -0,0 +1,104 @@
+package root
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/aerogear/charmil/validator/rules"
+	"github.com/redhat-developer/app-services-cli/internal/build"
+	"github.com/redhat-developer/app-services-cli/internal/config"
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/factory"
+	"github.com/redhat-developer/app-services-cli/pkg/localize/goi18n"
+)
+
+func Test_ExecuteCommand(t *testing.T) {
+
+	localizer, err := goi18n.New(nil)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+
+	buildVersion := build.Version
+	cmdFactory := factory.New(build.Version, localizer)
+	if err != nil {
+		fmt.Println(cmdFactory.IOStreams.ErrOut, err)
+		os.Exit(1)
+	}
+
+	initConfig(cmdFactory)
+
+	cmd := NewRootCommand(cmdFactory, buildVersion)
+
+	// Testing cobra commands with default recommended config
+	var vali rules.RuleConfig
+	validationErr := vali.ExecuteRules(cmd)
+	for _, errs := range validationErr {
+		if errs.Err != nil {
+			t.Errorf("%s: cmd %s: %s", errs.Rule, errs.Cmd.CommandPath(), errs.Name)
+		}
+	}
+
+	if len(validationErr) == 0 {
```

Comment:
Actually I have idea. How about we do keep test passing but have commented out assertions and follow up issue.
That will be better than changing rules. If we can make rules more adjusted to existing commands in rhoas + keep asertions to detect new commands but ignore errors that we currently have tat will be great deal.